test/controllers/drivers_controller_test.rb

```diff
@@ -6,68 +6,110 @@
   describe "index" do
     it "responds with success when there are many drivers saved" do
       # Arrange
+      Driver.create(name: "Kari", vin: "IHNXZOJKN4H52RI3Z", available: "true")
+      Driver.create(name: "Kendrick Marks Jr", vin: "JDO9OQL7AWDLZ1AY6", available: "false")
+
       # Ensure that there is at least one Driver saved
+      expect(Driver.all).wont_be_empty
+      expect(Driver.all).wont_be_nil
 
       # Act
+      get drivers_path
 
       # Assert
+      must_respond_with :success
 
     end
 
     it "responds with success when there are no drivers saved" do
       # Arrange
       # Ensure that there are zero drivers saved
+      Driver.destroy_all
+      expect(Driver.all.length).must_equal 0
 
       # Act
+      get drivers_path
 
       # Assert
+      must_respond_with :success
 
     end
   end
 
   describe "show" do
+    before do
+      @driver = Driver.create(name: "Kendrick Marks Jr", vin: "IHNXZOJKN6H52RI3Z", available: "false")
+    end
     it "responds with success when showing an existing valid driver" do
       # Arrange
       # Ensure that there is a driver saved
 
+      valid_driver_id = @driver.id
+
       # Act
+      get driver_path(valid_driver_id)
 
       # Assert
+      must_respond_with :success
 
     end
 
-    it "responds with 404 with an invalid driver id" do
+    it "will redirect for an invalid driver id" do
+
       # Arrange
-      # Ensure that there is an id that points to no driver
+      invalid_driver_id = -1
 
       # Act
+      expect(Driver.find_by(id: invalid_driver_id)).must_be_nil
```

Comment:
I like verifying that your assumptions are true.  I'd probably put the `expect` into the "arrange" section like you did above though.